submodules/groups/groups.js

```diff
@@ -1054,9 +1056,24 @@ define(function(require) {
 							});
 						},
 						function(callback) {
-							self.groupsUpdateCallflow(data.callflow, function(data) {
+							if (baseGroupName !== data.callflow.name) {
```

Comment:
Here you can use a guard to get ride of this `if/else`:
```suggestion
							if (baseGroupName === data.callflow.name) {
								callback(null);
								return;
							}
```

Reply:
done